Helper/Data.php

```diff
@@ -613,10 +613,18 @@ public function getHmac($storeId = null)
     {
         switch ($this->isDemoMode($storeId)) {
             case true:
-                $secretWord = $this->_encryptor->decrypt(trim($this->getAdyenHppConfigData('hmac_test', $storeId)));
+                $hmacTest = $this->getAdyenHppConfigData('hmac_test', $storeId);
+                if (is_null($hmacTest)) {
+                    return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
                    return null;
```